docs/mixed_precision.md

```diff
@@ -0,0 +1,35 @@
+# 混合精度训练
+
+## 简介
+PLSC支持混合精度训练。使用混合精度训练可以提升训练的速度，同时减少训练使用的显存开销。
+
+## 使用方法
+可以通过下面的代码设置开启混合精度训练：
+
+```python
+from __future__ import print_function
+import plsc.entry as entry
```

Comment:
Now we can import Entry from plsc directly, e.g., from plsc import Entry

Reply:
done.thanks!